### PR TITLE
Annotations doc release note changes for 2.1

### PR DESCRIPTION
**Pull request by scottmarlow**
Signed-off-by: Scott Marlow <smarlow@redhat.com>

### CODE REVIEW

File: install/caj/docs/CAJ2.1-ReleaseNotes.html [hunk 1]
```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<html>
+  <head>
+    <meta http-equiv="content-type" content="text/html; charset=windows-1252">
+    <title>Jakarta Annotations TCK, Version 2.0 Release Notes, September 2020</title>
```
Comment (gurunrao) [inline install/caj/docs/CAJ2.1-ReleaseNotes.html:5]: 2.1 release note.

Comment (scottmarlow) [reply install/caj/docs/CAJ2.1-ReleaseNotes.html]: fixed

File: install/caj/docs/CAJ2.1-ReleaseNotes.html [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+      </li>
+    </ul>
+    <hr>
+    <h2><a name="platform">Platform Notes</a></h2>
+    <p>The Jakarta Annotations TCK tests have been built with JDK 11 and tested
+      with OpenJDK 11 and OpenJDK 17.</p>
+    <p>The Jakarta Annotations TCK tests have been run on the following
+      platforms:</p>
+    <ul>
+      <li>TBD: include OS + version</li>
```
Comment (gurunrao) [inline install/caj/docs/CAJ2.1-ReleaseNotes.html:51]: OS as we discussed at PR https://github.com/eclipse-ee4j/jakartaee-tck/pull/788#discussion_r781793098

Comment (scottmarlow) [reply install/caj/docs/CAJ2.1-ReleaseNotes.html]: Switched OS to `Debian`

**Review by gurunrao — APPROVED** [review]

**Pull request merged**